src/client/components/footer.js

```diff
@@ -44,6 +44,7 @@ function Footer(props) {
 									alt="BrowserStack Logo"
 									height="25"
 									src="/images/BrowserStack.png"
+									width="95"
```

Comment:
This distorts the image, which doesn't look very good.
Can you find another solution for this (or also change the height accordingly?)